compiler/rustc_mir/src/borrow_check/region_infer/values.rs

```diff
@@ -114,7 +114,7 @@ rustc_index::newtype_index! {
 
 /// An individual element in a region value -- the value of a
 /// particular region variable consists of a set of these elements.
-#[derive(Debug, Clone)]
+#[derive(Debug, Copy, Clone)]
```

Comment:
Are there many places where `RegionElement` being `Copy` cleans up your code? I can't think of a case for this type in particular, but I am always weary of adding `derive(Copy)` to types, regardless of size because it might change semantics that other code relies on. This _seems_ safe though.

Reply:
Not many places at all no, I can remove that

Reply:
I've rebased and pushed a commit to address this